native/src/main/java/io/ballerina/stdlib/ftp/server/FtpListenerHelper.java

```diff
@@ -128,8 +128,11 @@ private static Map<String, String> getServerConnectorParamMap(BMap serviceEndpoi
                 final String privateKeyPath = (privateKey.getStringValue(StringUtils.fromString(
                         FtpConstants.ENDPOINT_CONFIG_KEY_PATH))).getValue();
```

Comment:
Shall we do the same empty check for `privateKeyPath` as well and return an error to the user if it is null/empty?

Reply:
since `path` is a required parameter we don't need to check for null right?
Added a check for empty key path in https://github.com/ballerina-platform/module-ballerina-ftp/pull/624/commits/acea76bb7a8f27da25280dc42b0145dcc794e5d6

Reply:
Yes, Checking if the string is empty would be enough.